### PR TITLE
INVGOV-13388 /locations/ missing Node.js API endpoint

### PR DESCRIPTION
**Pull request by sam-ludlow**
https://io1dev.atlassian.net/browse/INVGOV-13388

### CODE REVIEW

File: api.swagger.yaml [hunk 1]
```diff
@@ -6657,6 +6657,89 @@ paths:
         - SessionCookieHeader: []
         - APIKeyHeader: []
         - CognitoJwt: [ invotra/locations:write ]
+  '/locations/any/{location_id}':
```
Comment (ParryLuke) [inline api.swagger.yaml:6660]: Maybe we just go with /locations/id on this one?

File: api.swagger.yaml [hunk 2]
```diff
@@ -17659,6 +17758,19 @@ components:
           description: The location's email address.
           type: string
           format: email
+    location-any-schema:
+      oneOf:
```
Comment (ParryLuke) [inline api.swagger.yaml:17762]: https://dev.invotra.com/?url=https://raw.githubusercontent.com/invotra/api/ca1474c26ae66b7336b8943e1dda01d9775d65af/api.swagger.yaml

(no example available) showing, is it possible to just return the first schema?

File: api.swagger.yaml [hunk 3]
```diff
@@ -6657,6 +6657,89 @@ paths:
...
+        '403':
+          description: The user does not have permission to view this location.
+        '404':
+          description: The UUID does not correspond to an existing location.
+      description: Get information about the location.
+      security:
+        - SessionCookieHeader: []
+        - APIKeyHeader: []
+        - CognitoJwt: [ invotra/locations:read ]
+  '/locations/{location_id}/members':
```
Comment (ParryLuke) [inline api.swagger.yaml:6690]: Any risks for solr here / new data?

Some dbs will have a lot of users in certain locations

**Review by ParryLuke — CHANGES_REQUESTED** [review]

File: api.swagger.yaml (hunk 1 above)
Comment (sam-ludlow) [reply api.swagger.yaml]: makes sense, done.

File: api.swagger.yaml (hunk 2 above)
Comment (sam-ludlow) [reply api.swagger.yaml:19787]: I think our version of swagger UI needs a bump. It looks fine in editor and doign this:
docker pull swaggerapi/swagger-ui
docker run -p 8080:8080 swaggerapi/swagger-ui

http://127.0.0.1:8080/
https://dev.invotra.com/?url=https://raw.githubusercontent.com/invotra/api/INVGOV-13388.SWAGGER-5.17/api.swagger.yaml

File: api.swagger.yaml (hunk 3 above)
Comment (sam-ludlow) [reply api.swagger.yaml:7947]: This wont be a problem, no additional solr indexing, will just page through users index with a filter on location TID.

**Review by ParryLuke — APPROVED** [review]

**Pull request merged**